coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -2366,6 +2368,26 @@ def forward(self, x):
 
         self.run_compare_torch(shape, OnesStaticModel().eval(), backend=backend)
 
+class TestRandint(TorchBaseTest):
+    @pytest.mark.parametrize(
+        "backend, shape, low, high",
+        itertools.product(
+            backends,
+            [(1,), (2, 3)],
+            [-1, 2],
+            [3, 5],
+        ),
+    )
+    def test_randint(self, backend, shape, low, high):
+        class Randint(nn.Module):
+            def forward(self, x):
+                return torch.randint(low, high, x.shape)
```

Comment:
Something we have done before in our unit tests when the model is nondeterministic [is to just return the length](https://github.com/apple/coremltools/blob/f7fba76f15295719c7ec9e5dddefcd3973b74c03/coremltools/converters/mil/frontend/torch/test/test_torch_ops.py#L1488). This allows us to use our existing testing helper functions.